test/testdefs.jl

```diff
@@ -1,5 +1,22 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+if haskey(ENV, "JULIA_TEST_EXEFLAGS")
+    const test_exeflags = `$(Base.shell_split(ENV["JULIA_TEST_EXEFLAGS"]))`
+else
+    inline_flag = Base.JLOptions().can_inline == 1 ? `` : `--inline=no`
+    cov_flag = ``
+    if Base.JLOptions().code_coverage == 1
+        cov_flag = `--code-coverage=user`
+    elseif Base.JLOptions().code_coverage == 2
+        cov_flag = `--code-coverage=all`
+    end
+    const test_exeflags = `$cov_flag $inline_flag --check-bounds=yes --startup-file=no --depwarn=error`
+end
+
+const test_exename = get(ENV, "JULIA_TEST_EXENAME", joinpath(JULIA_HOME, Base.julia_exename()))
```

Comment:
... Just realized that I didn't press the "submit" button.....

Yes, it should be reverted. There are two uses as I discribed in the pr that adds these. It is useful to replace one without having to specify the other.